ci_scripts/suppressions/lsan.supp

```diff
@@ -0,0 +1,15 @@
+leak:save_ps_display_args
+leak:initdb.c
+leak:fe_memutils.c
+leak:fe-exec.c
+leak:fe-connect.c
+leak:pqexpbuffer.c
+leak:strdup
+leak:preproc.y
+leak:pgtypeslib/common.c
+leak:ecpglib/memory.c
+leak:kmip.c
```

Comment:
can we create a ticket/tickets about this? Some of these supressions definitely seem like issues.

Reply:
Most of them are not real issues, but "design features" or just allocation functions. I used timescale as a reference and they mute the same things: https://github.com/timescale/timescaledb/blob/main/scripts/suppressions/suppr_leak.txt

But I'm agree that this list should be reviewed. Probably some mutes scope can be reduced. Here is the ticket: https://perconadev.atlassian.net/browse/PG-1588

Reply:
The `kmip.c` one is likely to be an actual bug. But, yeah, that can be outside the scope of this PR:

Reply:
Yes, and strdup also seems to be a possible issue, or at least it can hide possible issues, as it suppresses any leaked strings created by strdup - even if it's a leak in postgres itself, we should look at that.